projects/pointnav_baselines/experiments/habitat/pointnav_habitat_base.py

```diff
@@ -96,22 +127,55 @@ class PointNavHabitatBaseConfig(PointNavBaseConfig, ABC):
         HABITAT_CONFIGS_DIR, "tasks/pointnav_gibson.yaml"
     )
 
-    NUM_TRAIN_PROCESSES = max(5 * torch.cuda.device_count() - 1, 4)
-    NUM_VAL_PROCESSES = 1
-    NUM_TEST_PROCESSES = 10
+    ACTION_SPACE = gym.spaces.Discrete(len(PointNavTask.class_action_names()))
 
-    TRAINING_GPUS = list(range(torch.cuda.device_count()))
-    VALIDATION_GPUS = [torch.cuda.device_count() - 1]
-    TESTING_GPUS = [torch.cuda.device_count() - 1]
+    DEFAULT_NUM_TRAIN_PROCESSES = (
+        max(5 * torch.cuda.device_count(), 4) if torch.cuda.is_available() else 1
```

Comment:
Is this `max` operation doing anything? (I, perhaps wrongly, think it doesn't)

Reply:
Haha, yes you're right.